test/models/user_test.exs

```diff
@@ -0,0 +1,86 @@
+defmodule Yggdrasil.UserTest do
+  use Yggdrasil.ModelCase
+
+  alias Yggdrasil.User
+
+  @valid_attrs %{username: "tester", password: "password", password_confirmation: "password"}
```

Comment:
the valid_attrs for was something that was generated I just updated it and I stuck with that pattern. Seemed to be the convention and rolled with it.

Kind of on the fence, at first I was like the function are more attractive, but being able to just look in one place with the attribute lets you digest exactly what is passed in and it being at the top of the file instead of another file means you can simply scroll up and look at the details.

If there was more to creating the map I would certainly prefer the functions, so maybe worth while to adopt that pattern just to be consistent when those cases arise?